.github/workflows/top-issues-dashboard.yml

```diff
@@ -40,14 +40,10 @@ jobs:
         with:
           top_list_size: 10
           filter: "1772"
-          label: true
+          label: false
```

Comment:
@qwerty541, do you want the labels removed and only rely on the dashboard 🤔?

Reply:
> @qwerty541, do you want the labels removed and only rely on the dashboard 🤔?

@rickstaa Since we made a decision to pause addition of themes pull requests, i think would be good to remove "Top 'themes' pull requests" section from top issues dashboard. It is exactly what is trying to make in this pull request and nothing more. I have found your pull request https://github.com/anuraghazra/github-readme-stats/pull/2711 which added these section and reverted these changes. Honestly, i do not know if `label: true` string affect on something else, if it is we can keep it as is.

Reply:
Ah, I see 👍🏻. The `Top themes` label will be removed when the following lines will be removed:

```yml
custom_pull_requests_label: themes
top_custom_pull_requests_label: ":star: top themes"
top_custom_pull_requests_label_description: Top themes
top_custom_pull_requests_label_colour: "#A23599"
```

The `label: false` flag will disable all labels.